canonical/juju-dashboard
# WD-5818 - Syntax highlighting

## Done

- Add syntax highlighting to the JSON output.

## QA

- Go to the advanced search page and do a search.
- Switch to the JSON view.
- The JSON should have syntax highlighting.

## Details

https://warthogs.atlassian.net/browse/WD-5818

## Screenshots

<img width="1511" alt="Screenshot 2023-08-28 at 3 55 11 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/b0bd8ab2-68d1-4b3e-9875-1ad1e3783cdc">
